rts/Lua/LuaSyncedRead.cpp

```diff
@@ -1581,13 +1583,15 @@ int LuaSyncedRead::GetTeamStartPosition(lua_State* L)
 	if (!LuaUtils::IsAlliedTeam(L, team->teamNum))
 		return 0;
 
-	const float3& pos = team->GetStartPos();
+	const float4& pos = team->GetStartPos();
 
 	lua_pushnumber(L, pos.x);
 	lua_pushnumber(L, pos.y);
 	lua_pushnumber(L, pos.z);
+	lua_pushnumber(L, pos.w); // facing
```

Comment:
TODO: this is a potential backwards break, however, due to the documentation not being correct there is a chance that the previous 4th argument was never used. Needs to be researched.

Reply:
So yes, ZERO-K is using it, but also apparently it is bugged and they actually end up not using it a few lines later:
https://github.com/ZeroK-RTS/Zero-K/blob/master/LuaRules/Gadgets/init_auto_ready.lua#L75

so we might be better off removing the `hasValidStartPos` return